src/summary_statistics/tmrca.h

```diff
@@ -37,7 +37,7 @@ class TMRCA : public SummaryStatistic
    ~TMRCA() {};
 
    //Virtual methods
-   void calculate(const Forest &forest);
+   void calculate( Forest &forest);
```

Comment:
I don't remember the exact error, but it was a consequence of needing a non-constant forest in samplePoint. I had to do this to adjust the importance weight appropriately (particle.cpp ln 808 this->IS_TreePoint_adjustor( tp )). Once I made the argument for samplePoint non-constant I had to change the argument for all of these othere functions.